.ci/ci_cache_nemu.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# The main purpose of this script is to
+# optimise the nemu build from source and
+# use a cache as a prebuilt
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# Source to trap error line number
+source "${script_dir}/../lib/common.bash"
+
+WORKSPACE=${WORKSPACE:-$(pwd)}
+nemu_dir="/usr/local/bin/"
+
+cache_built_nemu() {
+	nemu_path="${1}"
+	real_path=$(readlink -f ${nemu_path})
+	nemu_binary=$(basename "${real_path}")
+	echo $(basename "${nemu_binary}") >  "latest"
+	cp "${real_path}" ./
```

Comment:
- Using `install` might be better or atleast `cp -a` to preserve perms?
- Why not just say `.`?